carbon-design-system/carbon
# docs(aspect-ratio): update aspect ratio docs for v11

This PR introduces two updates to our storybook for v11:

- Grid storybook styles are now scoped to a storybook-specific class name
- The Aspect Ratio story has been updated and includes controls

#### Changelog

**New**

- The AspectRatio story has been updated and now uses controls

**Changed**

- The Grid storybook styles are now scoped

**Removed**

#### Testing / Reviewing

- Verify no regression exists between the current grid storybook styles and the ones in this PR
- Verify the AspectRatio story renders as intended (correct aspect ratio and background color per item)
  - Also, verify the controls work when switching the "ratio" prop
